src/game/mario_step.c

```diff
@@ -259,10 +259,9 @@ s32 stationary_ground_step(struct MarioState *m) {
     if (takeStep) {
         stepResult = perform_ground_step(m);
     } else {
-        // Hackersm64: this condition fixes potential downwarps
-        if (m->pos[1] <= m->floorHeight + 160.0f) {
-            m->pos[1] = m->floorHeight;
-        }
+        //! TODO - This is responsible for many stationary downwarps but is
```

Comment:
scary

Reply:
not even scary this is just vanilla 😔 whats scary is massive untested vanilla behavior changes

Reply:
@arthurtilly this results in jank itself so I think a solution should be found, you don't run into it in vanilla as much as you do in hacks

Reply:
where do you run into it in hacks? i cant think of any common downwarps

Reply:
I've run into the downwarp multiple different times in DimensiOne. It's actually pretty common.

Reply:
that downwarp is different and not really caused by this code, the problem is the action getting changed to a grounded one in the first place when it shouldnt. should be fixed yeah but this isnt a good solution